admin-dev/themes/new-theme/js/app/utils/column-toggling.js

```diff
@@ -31,40 +31,48 @@ const $ = global.$;
  */
 class ColumnToggling {
```

Comment:
`admin-dev/themes/new-theme/js/app/utils/column-toggling.js` is not the right place for this file, everything under `admin-dev/themes/new-theme/js/app` is Vue related stuff.